packages/p2p/stores/order-details-store.js

```diff
@@ -0,0 +1,96 @@
+import { observable, action } from 'mobx';
+import { requestWS } from 'Utils/websocket';
+import { secondsToTimer } from 'Utils/date-time';
+import ServerTime from 'Utils/server-time';
+import { localize } from 'Components/i18next';
+
+export default class OrderDetailsStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        this.general_store = this.root_store.general_store;
+    }
+
+    @observable channel_url = this.root_store.order_store.order_information?.chat_channel_url;
+    @observable interval = null;
+    @observable popup_options = {};
+    @observable remaining_time;
+    @observable should_show_popup = false;
+
+    @action.bound
+    clearIntervalState() {
+        clearInterval(this.interval);
+    }
```

Comment:
Let's take care of this in the component rather than here.

Reply:
Done. Thank you for reviewing Aaron! 🙂